docs/content/Coding Milestones/PoC2023q1/example1-subs/example1-stage-3.md

```diff
@@ -117,14 +117,13 @@ the following command checks that.
 test $(kubestellar-list-syncing-objects --api-group apps --api-kind ReplicaSet | grep "^ *kcp.io/cluster: [0-9a-z]*$" | sort | uniq | wc -l) -ge 2
 ```
 
-The various APIBinding and CustomResourceDefinition objects involved
+The CustomResourceDefinition objects involved
 should also appear in the mailbox workspaces.
 
 ```shell
 # TODO: kubestellar-list-syncing-objects has kcp dependencies. Will remove when controllers support spaces.
-test $(kubestellar-list-syncing-objects --api-group apis.kcp.io --api-version v1alpha1 --api-kind APIBinding | grep -cw "name: bind-apps") -ge 2
-kubestellar-list-syncing-objects --api-group apis.kcp.io --api-version v1alpha1 --api-kind APIBinding | grep -w "name: bind-kubernetes"
-kubestellar-list-syncing-objects --api-group apiextensions.k8s.io --api-kind CustomResourceDefinition | fgrep -w "name: crontabs.stable.example.com"
```

Comment:
Please do not remove the check for the user-defined CRD.

Reply:
Got it back in latest push.

Reply:
yes